_data/news.yml

```diff
@@ -1,3 +1,7 @@
+- name: spaghetti 1.3
+  date: 2019-05-22
+  summary: <a href="https://pysal-spaghetti.readthedocs.io/en/latest/">spaghetti 1.3</a> released on <a href="https://pypi.org/project/spaghetti/1.3/">PyPI</a> and <a https://anaconda.org/conda-forge/spaghetti">conda-forge</a>. This is the first version that officially supports Python 3.6 and 3.7 exclusively.
```

Comment:
![image](https://user-images.githubusercontent.com/7359284/58214922-72938380-7cac-11e9-8c3c-325c3fbd55cc.png)
Did not notice it. Could you correct it?

Reply:
Addressed in #101